TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,248 @@
+
+#### Global Semantics
+
+The semantics define global properties for all vector elements.
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `POLYGONS_LENGTH` | `uint32` | The number of pre-triangulated polygons in the tile. | :red_circle: No. |
+| `POLYLINES_LENGTH` | `uint32` | The number of polylines in the tile. | :red_circle: No. |
+| `POINTS_LENGTH` | `uint32` | The number of points in the tile. | :red_circle: No. |
+| `MESHES_LENGTH` | `uint32` | The number of meshes in the tile. | :red_circle: No. |
+| `BOXES_LENGTH` | `uint32` | The number of boxes in the tile. | :red_circle: No. |
+| `CYLINDERS_LENGTH` | `uint32` | The number of cylinders in the tile. | :red_circle: No. |
+| `ELLIPSOIDS_LENGTH` | `uint32` | The number of ellipsoids in the tile. | :red_circle: No. |
+| `SPHERES_LENGTH` | `uint32` | The number of spheres in the tile. | :red_circle: No. |
+| `MINIMUM_HEIGHT` | `float32` | The minimum height for this tiles' region in meters above the WGS84 ellipsoid. | :white_check_mark: Yes. |
+| `MAXIMUM_HEIGHT` | `float32` | The maximum height for this tiles' region in meters above the WGS84 ellipsoid. | :white_check_mark: Yes. |
+| `RECTANGLE` | `float32[]` | The rectangle containing the geometry in the tile. It is an array with the four elements west, south, east, and north (in that order). | :white_check_mark: Yes. |
+| `RTC_CENTER` | `float32[]` | The center used for RTC rendering. | :red_circle: No. If no center is supplied, the center of the global `RECTANGLE` will be used. |
+
+#### Vector Semantics
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `POLYGON_COUNTS` | `uint32[]` | The number of vertices that belong to each polygon. This refers to the polygon section of the positions buffer in the body. Each polygon count refers to a contiguous number of vertices in the position buffer that represents the polygon.  | :white_check_mark: Yes, when the global `POLYGONS_LENGTH` is greater than zero. |
+| `POLYGON_INDEX_COUNTS` | `uint32[]` | The number of indices that belong to each polygon. This refers to the indices buffer of the body. Each index count refers to a contiguous number of indices that represent the triangulated polygon. | :white_check_mark: Yes, when the global `POLYGONS_LENGTH` is greater than zero. |
+| `POLYGON_MINIMUM_HEIGHTS` | `float32[]` | The minimum height of each polygon in meters above the WGS84 ellipsoid. | :red_circle: No. If the minimum height for each polygon is not specified, the global `MINIMUM_HEIGHT` will be used. |
+| `POLYGON_MAXIMUM_HEIGHTS` | `float32[]` | The maximum height of each polygon in meters above the WGS84 ellipsoid. | :red_circle: No. If the maximum height for each polygon is not specified, the global `MAXIMUM_HEIGHT` will be used. |
+| `POLYGON_BATCH_IDS` | `uint16[]` | The `batchId` of the polygon that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
```

Comment:
Are these often in increasing order?  It is worth having a fast path where if these are not defined, it is assume to be auto-incrementing from zero with a defined order of each feature type?